e2e/tests/nightly/gke-apps/cromwell.spec.ts

```diff
@@ -16,6 +16,10 @@ const fileBasePath = '../../../../resources/cromwell/';
 const wdlFilePath = path.relative(process.cwd(), __dirname + fileBasePath + wdlFileName);
 const jsonFilePath = path.relative(process.cwd(), __dirname + fileBasePath + jsonFileName);
 
+const cromshellSubmitPythonCmd = `!%s submit ${wdlFileName} ${jsonFileName}`;
+const cromshellSubmitRCmd =
+  `system2('%s', args = c('submit', '${wdlFileName}','${jsonFileName}'), ` + ' stdout = TRUE, stderr = TRUE)';
```

Comment:
Minor: I find the combined use of string concatenation, backtick-syntax, and `%s` confusing here.

Maybe these could be functions?

```
const cromshellSubmitRCmd = cromshell_version =>
  `system2('${cromshell_version}', args = c('submit', '${wdlFileName}','${jsonFileName}'), stdout = TRUE, stderr = TRUE)`;
```